Source/driver/Castro.cpp

```diff
@@ -1473,7 +1473,7 @@ Castro::estTimeStep (Real dt_old)
 #endif
 
     if (verbose) {
-        amrex::Print() << "Castro::estTimeStep (" << limiter << "-limited) at level " << level << ":  estdt = " << estdt << '\n';
+        amrex::Print() << "Castro::estTimeStep (" << limiter << "-limited) at level " << level << ":  estdt = " << estdt << '\n' << std::endl;
```

Comment:
do you need both `\n` and `std::endl`?

Reply:
My general approach has been that most things should add two newlines so the stdout is easier to read.